webapp/src/assets/styles/index.css

```diff
@@ -62,6 +59,6 @@
   transform: translate(-50%, 2%);
 }
 
-.loading {
```

Comment:
Remember that classes are global, they need to be appropriate in any context. If you are doing component specific classes, you need to use css modules, or prefix the classes fully.